docs/ts/primitives/streaming-apis.mdx

```diff
@@ -8,118 +8,269 @@ lang: ts
 
 Encore makes it easy to create API endpoints that can stream data to and from your applications.
 
+<GitHubLink
+    href="https://github.com/encoredev/examples/tree/main/ts/streaming-chat"
+    desc="Simple chat app using the Streaming API create a WebSocket stream from a web frontend."
+/>
+
 ## Different kinds of stream
 
 Encore supports three types of streams, each designed for a specific data flow direction:
-- **StreamIn**: When you need to stream data into your service.
-- **StreamOut**: When you need to stream data out from your service.
-- **StreamInOut**: When you need to stream data into and out of service.
+- [**StreamIn**](#streamin): When you need to stream data into your service.
+- [**StreamOut**](#streamout): When you need to stream data out from your service.
+- [**StreamInOut**](#streaminout): When you need to stream data into and out of service.
```

Comment:
```suggestion
- [**StreamInOut**](#streaminout): When you need to stream data into and out of your service.
```
saw that I had missed adding a word here

Reply:
Thanks! I have pushed a new commit